guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>This model will train for 100 epochs using the provided learning rate. If the learning rate is not mentioned, then an optimal learning rate is automatically deduced for training the model. </p><p>Other important inputs are the two optional boolean parameters of <code>early_stopping</code> and <code>checkpoint</code>. The <code>early_stopping</code> parameter by default is set to <code>False</code>, but if set to <code>True</code> , will stop the training if validation loss stops improving for 5 epochs. The <code>checkpoint</code> parameter can be used to save the best model during training. <s>which if</s> <u>When </u>set to <code>True</code>, the best model based on validation loss will be saved during training.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>